cmd/pinniped-proxy/src/cli.rs

```diff
@@ -13,33 +13,44 @@ use structopt::StructOpt;
 /// short-lived client certificates, before forwarding the request onwards
 /// to the destination k8s API server.
 pub struct Options {
-    #[structopt(
-        short = "p",
+    #[arg(
+        short = 'p',
```

Comment:
nitpick: is `'p'` preferred over `"p"` in rustfmt ?

Reply:
Actually a different type - `p` is a character type, whereas "p" is effectively a vector of characters, that happens to have a length of 1. I was forced to change this one here because they type had changed in the library so that the short version of the arg is only a single char, I assume :)